js/main.js

```diff
@@ -1,382 +0,0 @@
-// Weather App
-// by Roger Mullins
-
-// Object Definitions
-
-let stateObj = {
-  city : "",
-  zipCode: "",
-  tempK: 0,
-  tempF: 0,
-  tempC: 0,
-  condition: "",
-  imageURL: ""
-};
-
-let lookupHistoryObj = {
-  city: "",
-  zipCode: ""
-};
-
-let localStorageObj = {
-  city: "",
-  zipCode: ""
-}
-
-// Global Variable
-
-let zipIn = "";   // Used to validate ZIP
-let tempF = 0;    // Used to update temperature background color
-
-function buildUI() {
-  // Creates HTML elements
-
```

Comment:
While I think I understand why you'd define functions within other functions in the future this could all be in one file and we would want to separate them out into their own functions for reuse across the project. This might even be a good spot to think about this as a Class instead with methods. This is fine for now though.